### PR TITLE
chore(workspace): Bump rules_nodejs to 3.0.0, new Jest rule based on Wave

### PR DESCRIPTION
**Pull request by SkaillZ**
Fixes an issue with Jest tests on Windows.

#### Type of PR

Other

### CODE REVIEW

**Comment by SkaillZ** [comment]
Draft because the CI is acting up, local testing should already work.

File: tools/bazel_rules/jest/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (tomheller) [inline tools/bazel_rules/jest/.eslintrc.json:1]: We're not using eslint in barista. This is probably an artifact from copying it, right?
Can you make sure that linting is set up correctly for this project. 

With this, you should see the license header failing, as this is not yet changed in barista. 
I'm opening the PR right now.

**Review by tomheller — CHANGES_REQUESTED** [review]
Looks good, linting does not seem to be set up correctly though.

**Comment by SkaillZ** [comment]
Blocked by #1966

**Review by tomheller — APPROVED** [review]

**Review by tomheller — DISMISSED** [review]

**Review by lukasholzer — APPROVED** [review]

**Review by tomheller — APPROVED** [review]

**Pull request merged**